testbench/rest_server.py

```diff
@@ -1104,6 +1104,26 @@ def resumable_upload_chunk(bucket_name):
                 chunk_last_byte,
                 test_id,
             )
+
```

Comment:
I see that this logic is added specifically for resumable uploads. We need to cover single shot as well for Go. Maybe in the object_insert function above?

Reply:
Added support for this also for stalls. Although please note that other error codes are not supporting this feature. If you want I can add that support in further PRs.

Reply:
I did experiment on `return-503-after-3072K`. It was not throwing any googleapi error until I was uploading data less than chunk size(single shot upload). may be we can discuss offline about this.